examples/FsCheck.XUnit.CSharpExamples/ReverseFixture.cs

```diff
@@ -6,17 +6,25 @@ namespace FsCheck.XUnit.CSharpExamples
 {
     public class ReverseFixture
     {
-        //[Property]
-        //public void Bcl2()
-        //{
-        //}
-
-        [Property]
+        [Property(QuietOnSuccess = true)]
         public bool Bcl(int[] xs)
         {
           return xs.Reverse().Reverse().SequenceEqual(xs);
         }
 
+        [Property(QuietOnSuccess = true)]
+        public void Bcl2(int[] xs)
```

Comment:
I was using it to determine what the C# void return would do with TestDriven.NET. If you think I should remove it, I can do that.